internal/ent/entconfig/config.go

```diff
@@ -4,4 +4,76 @@ package entconfig
 type Config struct {
 	// EntityTypes is the list of entity types to create by default for the organization
 	EntityTypes []string `json:"entityTypes" koanf:"entityTypes" default:"" description:"entity types to create for the organization"`
+
+	Summarizer Summarizer `json:"summarizer" koanf:"summarizer"`
+}
+
+type Summarizer struct {
```

Comment:
You're going to want to include comments on all the exported structs in the format of // StructName is...

Reply:
Will do that now

Reply:
https://github.com/theopenlane/core/pull/689/commits/52ae9321f7c0b5922497c2100cd023bb5cbb6b58